### DISCUSSION TITLE
local tools to build Directus

### DISCUSSION THREAD

**Discussion by miiitch** [directus/directus#6388] [Archive]
Hi,

According to the doc (https://docs.directus.io/contributing/running-locally) only Node is needed. It looks like I need other tools. Right now, I had to install lerna and typescript and get some build error about fs-extra. Where can I find a comprehensive list of needed third party tools? 

FYI, I'm trying to build Directus from codespace.

Regards,

Michel

**Comment by rijkvanzanten** [comment]
You need the latest version of node (16 currently) and then a simple npm install in the root folder will suffice. That'll pull in all the dependencies required, including Lerna and TypeScript

**Reply by miiitch** [reply]
Thanks, I read latest but I took the LTS version. It's fine now
